tests/backup/backup_delete_test.go

```diff
@@ -467,3 +468,238 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 	})
 })
+
+// DeleteBucketVerifyCloudBackupMissing validates the backup state (CloudBackupMissing) when bucket is deleted.
+var _ = Describe("{DeleteBucketVerifyCloudBackupMissing}", func() {
+	var (
+		contexts                   []*scheduler.Context
+		appContexts                []*scheduler.Context
+		clusterUid                 string
+		cloudAccountUID            string
+		cloudAccountName           string
+		bkpLocationName            string
+		backupLocationUID          string
+		backupLocationMap          map[string]string
+		periodicSchedulePolicyName string
+		periodicSchedulePolicyUid  string
+		scheduleName               string
+		appNamespaces              []string
+		scheduleNames              []string
+		backupNames                []string
+		localBucketNameMap         map[string]string
+	)
+
+	providers := getProviders()
+	backupLocationMap = make(map[string]string)
+	localBucketNameMap = make(map[string]string)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("DeleteBucketVerifyCloudBackupMissing", "Validates the backup state (CloudBackupMissing) when bucket is deleted.", nil, 58070)
+		log.Infof("Deploying applications required for the testcase")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				appNamespaces = append(appNamespaces, namespace)
+			}
+		}
+	})
+
+	It("Delete bucket and Validates the backup state", func() {
+		Step("Validate deployed applications", func() {
+			ValidateApplications(contexts)
+		})
+
+		Step("Adding cloud path/bucket", func() {
+			log.InfoD("Adding cloud path/bucket")
+			for _, provider := range providers {
+				bucketNameSuffix := getBucketNameSuffix()
+				bucketNamePrefix := fmt.Sprintf("local-%s", provider)
+				localBucketName := fmt.Sprintf("%s-%s", bucketNamePrefix, bucketNameSuffix)
+				CreateBucket(provider, localBucketName)
```

Comment:
Why do you want a special bucket here?

Reply:
As per your comment, it is better to keep local bucket than use a global bucket, it will be less error prone.